README.md

```diff
@@ -452,6 +452,23 @@ In your vault go to Settings > Community plugins > Browse and search for "Execut
 or
 
 Follow [this link](https://obsidian.md/plugins?search=execute%20code#) and click "Open in Obsidian".
+	
+## Locating Path Settings ( ex. JavaScript | Node )
+	
+	To avoid or resolve errors from an incorrect path.

```

Comment:
I just noticed something else: since you have indentation before each line, your changes are all inside a code block. Was that intentional? sorry

Reply:
Thanks -- first pull request.  Now it looks right. :)

Reply:
gotcha :) no worries!!! PRs are always welcome, and thank you *very* much for working with us to make sure it's the best it can be!!